README.md

```diff
@@ -0,0 +1,106 @@
+# GroupDocs.Conversion for .NET MVC Example
+###### version 0.1.0
+
+[![Build status](https://ci.appveyor.com/api/projects/status/boo4pnp61r6b8kqp/branch/master?svg=true)](https://ci.appveyor.com/project/egorovpavel/groupdocs-conversion-for-net-mvc/branch/master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/dbaad4274f364d91929418a721ae2a45)](https://www.codacy.com/app/GroupDocs/GroupDocs.Conversion-for-.NET-MVC?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC&amp;utm_campaign=Badge_Grade)
```

Comment:
[![Codacy Badge](https://api.codacy.com/project/badge/Grade/2b18e93ff07c440a98b70dd84f1b6ebb)](https://www.codacy.com/app/GroupDocs/GroupDocs.Conversion-for-.NET-MVC?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC&amp;utm_campaign=Badge_Grade)